content/en/logs/log_collection/javascript.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Browser Log Collection
+kind: documentation
+aliases:
+  - /logs/log_collection/web_browser
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+- link: "logs/processing/parsing"
+  tag: "Documentation"
+  text: "Learn more about parsing"
+- link: "logs/explorer"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "logs/explorer/analytics"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "logs/faq/log-collection-troubleshooting-guide"
+  tag: "FAQ"
+  text: "Log Collection Troubleshooting Guide"
+---
+
+
+
+Send logs to Datadog from web browsers or other Javascript client thanks to the Datadog's `datadog-logs` client-side JavaScript logging library.
+
+With the `datadog-logs` library, you can send log directories to Datadog from JS clients and leverage the following features:
+
+* Use the library as a logger. Everything is forwarded to Datadog as JSON documents.
+* Add `context` and extra custom attributes to each log sent.
+* Wrap and forward every JavaScript error automatically.
+* Forward JavaScript's console logs.
+* Record real client IP addresses and user agents.
+* Optimised network usage with automatic bulk posts.
+
+## Get a Public API Key
+
+<div class="alert alert-warning">
+Public API keys are in private beta. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> to turn on this feature for your account.
+</div>
+
+For security reasons, [API keys][2] cannot be used to configure the `datadog-logs` library as they would be exposed client-side in the JavaScript code. To collect logs from web browsers, a [public API key][3] must be used.
```

Comment:
configure seems like the wrong word here. I assume you can't use API keys at all for any purpose, the limitation ins't specific to configuration @NBParis